fuzzers/072-ordered_wires/run_fuzzer.py

```diff
@@ -0,0 +1,49 @@
+import threading
+import os
+import shutil
+
+def start_vivado(blockID, start, stop):
+    print ("Running instance :" + str(blockID))
+    os.system("${XRAY_VIVADO} -mode batch -source $FUZDIR/job.tcl -tclargs " + str(blockID) + " " + str(start) + " " + str(stop))
+
+def get_nb_pips():
+    print ("Fetching nb pips")
+    os.system("${XRAY_VIVADO} -mode batch -source $FUZDIR/get_pipscount.tcl")
+    countfile = open("nb_pips.txt", "r")
+    return int(countfile.readline())
+
+if __name__ == "__main__":
```

Comment:
You would normally do something like the following;
```python
import sys

def main(argv):
   print("My code goes here")
   if len(argv) > 2:
      return 0
   else:
      return 1

if __name__ == "__main__":
  sys.exit(main(sys.argv))

Reply:
Oky will add that to the evening cleanup of that code :) Again I’m little rusty in python so next PR should be easier.